README.md

```diff
@@ -54,4 +54,4 @@ Check out these projects that build on top of runwasi:
 
 ## Contributing
 
-To begin contributing, please read our [Contributing Guide](https://runwasi.dev/developer/contributing.html).
+To begin contributing, please read our [Contributing Guide](https://runwasi.dev/CONTRIBUTING.html).
```

Comment:
I think anything based on checking runwasi.dev will fail only after runwasi.dev is deployed.

Reply:
How about a two phase linking strategy? 

1. Run link checkers for every PR, skip checking links to `runwasi.dev` or do not fail the PRs if there are errors related to links not found on `runwasi.dev`
2. Run another link checker for every deployment of the `runwasi.dev` and make sure that the links are working in the livesite.

Reply:
To elaborate more on the second point, I've added a action to create a GitHub issue for broken links it finds at post-deployment of runwasi.dev.